## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 506396`
Commit: [`9d7dcee3548d49f1268116871f22d5b38d454a1a`](https://github.com/NixOS/nixpkgs/commit/9d7dcee3548d49f1268116871f22d5b38d454a1a) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/9d7dcee3548d49f1268116871f22d5b38d454a1a..pull/506396/head))
Merge: [`bd8bf85336bb15db8caea3fb1f8e1b94716680dc`](https://github.com/NixOS/nixpkgs/commit/bd8bf85336bb15db8caea3fb1f8e1b94716680dc)

Logs: https://github.com/drupol/nixpkgs-review-gha/actions/runs/23961619940


---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>brioche</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>brioche</summary>
<pre>          /nix/store/rfp8lhk4dl9syfn64rwb3h3c73426p08-binutils-2.44/bin/ld: /build/source/target/x86_64-unknown-linux-gnu/release/deps/libv8-ef6ce659bea65e87.rlib(js-temporal-objects.o): in function `temporal_rs::Instant::to_ixdtf_string_with_compiled_data(temporal_rs::TimeZone const*, temporal_rs::ToStringRoundingOptions) const&#x27;:
          ./../../../../third_party/rust/chromium_crates_io/vendor/temporal_capi-v0_0_11/bindings/cpp/temporal_rs/Instant.hpp:171:(.text[_ZNK11temporal_rs7Instant34to_ixdtf_string_with_compiled_dataEPKNS_8TimeZoneENS_23ToStringRoundingOptionsE]+0xcf): undefined reference to `temporal_rs_Instant_to_ixdtf_string_with_compiled_data&#x27;
          /nix/store/rfp8lhk4dl9syfn64rwb3h3c73426p08-binutils-2.44/bin/ld: /build/source/target/x86_64-unknown-linux-gnu/release/deps/libv8-ef6ce659bea65e87.rlib(js-temporal-objects.o): in function `decltype(auto) std::__Cr::__variant_detail::__visitation::__base::__dispatcher&lt;0ul&gt;::__dispatch&lt;std::__Cr::__variant_detail::__dtor&lt;std::__Cr::__variant_detail::__traits&lt;diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::TimeZone, std::__Cr::default_delete&lt;temporal_rs::TimeZone&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;, (std::__Cr::__variant_detail::_Trait)1&gt;::__destroy()::{lambda(auto:1&amp;)#1}&amp;&amp;, std::__Cr::__variant_detail::__base&lt;(std::__Cr::__variant_detail::_Trait)1, diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::TimeZone, std::__Cr::default_delete&lt;temporal_rs::TimeZone&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;&amp;&gt;(std::__Cr::__variant_detail::__dtor&lt;std::__Cr::__variant_detail::__traits&lt;diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::TimeZone, std::__Cr::default_delete&lt;temporal_rs::TimeZone&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;, (std::__Cr::__variant_detail::_Trait)1&gt;::__destroy()::{lambda(auto:1&amp;)#1}&amp;&amp;, std::__Cr::__variant_detail::__base&lt;(std::__Cr::__variant_detail::_Trait)1, diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::TimeZone, std::__Cr::default_delete&lt;temporal_rs::TimeZone&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;&amp;)&#x27;:
          ./../../../../third_party/libc++/src/include/variant:534:(.text[_ZNSt4__Cr16__variant_detail12__visitation6__base12__dispatcherIJLm0EEE10__dispatchIOZNS0_6__dtorINS0_8__traitsIJN8diplomat2OkINS_10unique_ptrIN11temporal_rs8TimeZoneENS_14default_deleteISC_EEEEEENS8_3ErrINSB_13TemporalErrorEEEEEELNS0_6_TraitE1EE9__destroyEvEUlRT_E_JRNS0_6__baseILSL_1EJSG_SJ_EEEEEEDcSN_DpT0_]+0x15): undefined reference to `temporal_rs_TimeZone_destroy&#x27;
          /nix/store/rfp8lhk4dl9syfn64rwb3h3c73426p08-binutils-2.44/bin/ld: /build/source/target/x86_64-unknown-linux-gnu/release/deps/libv8-ef6ce659bea65e87.rlib(js-temporal-objects.o): in function `decltype(auto) std::__Cr::__variant_detail::__visitation::__base::__dispatcher&lt;0ul&gt;::__dispatch&lt;std::__Cr::__variant_detail::__dtor&lt;std::__Cr::__variant_detail::__traits&lt;diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;, (std::__Cr::__variant_detail::_Trait)1&gt;::__destroy()::{lambda(auto:1&amp;)#1}&amp;&amp;, std::__Cr::__variant_detail::__base&lt;(std::__Cr::__variant_detail::_Trait)1, diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;&amp;&gt;(std::__Cr::__variant_detail::__dtor&lt;std::__Cr::__variant_detail::__traits&lt;diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;, (std::__Cr::__variant_detail::_Trait)1&gt;::__destroy()::{lambda(auto:1&amp;)#1}&amp;&amp;, std::__Cr::__variant_detail::__base&lt;(std::__Cr::__variant_detail::_Trait)1, diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;&amp;)&#x27;:
          ./../../../../third_party/libc++/src/include/variant:534:(.text[_ZNSt4__Cr16__variant_detail12__visitation6__base12__dispatcherIJLm0EEE10__dispatchIOZNS0_6__dtorINS0_8__traitsIJN8diplomat2OkINS_10unique_ptrIN11temporal_rs25OwnedPartialZonedDateTimeENS_14default_deleteISC_EEEEEENS8_3ErrINSB_13TemporalErrorEEEEEELNS0_6_TraitE1EE9__destroyEvEUlRT_E_JRNS0_6__baseILSL_1EJSG_SJ_EEEEEEDcSN_DpT0_]+0x15): undefined reference to `temporal_rs_OwnedPartialZonedDateTime_destroy&#x27;
          /nix/store/rfp8lhk4dl9syfn64rwb3h3c73426p08-binutils-2.44/bin/ld: /build/source/target/x86_64-unknown-linux-gnu/release/deps/libv8-ef6ce659bea65e87.rlib(js-temporal-objects.o): in function `v8::internal::temporal::CombinedRecord::operator=(v8::internal::temporal::CombinedRecord&amp;&amp;)&#x27;:
          ./../../../../v8/src/objects/js-temporal-objects.cc:1711:(.text[_ZN2v88internal8temporal14CombinedRecordaSEOS2_]+0xf1): undefined reference to `temporal_rs_TimeZone_destroy&#x27;
          /nix/store/rfp8lhk4dl9syfn64rwb3h3c73426p08-binutils-2.44/bin/ld: ./../../../../v8/src/objects/js-temporal-objects.cc:1711:(.text[_ZN2v88internal8temporal14CombinedRecordaSEOS2_]+0x155): undefined reference to `temporal_rs_TimeZone_destroy&#x27;
          /nix/store/rfp8lhk4dl9syfn64rwb3h3c73426p08-binutils-2.44/bin/ld: /build/source/target/x86_64-unknown-linux-gnu/release/deps/libv8-ef6ce659bea65e87.rlib(js-temporal-objects.o): in function `diplomat::result&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt;, temporal_rs::TemporalError&gt; std::__Cr::__function::__policy_func&lt;diplomat::result&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt;, temporal_rs::TemporalError&gt; (std::__Cr::basic_string_view&lt;char, std::__Cr::char_traits&lt;char&gt; &gt;)&gt;::__call_func&lt;v8::internal::temporal::ToTemporalZonedDateTime(v8::internal::Isolate*, v8::internal::DirectHandle&lt;v8::internal::Object&gt;, v8::internal::MaybeDirectHandle&lt;v8::internal::Object&gt;, char const*)::$_0&gt;(std::__Cr::__function::__policy_storage const*, std::__Cr::basic_string_view&lt;char, std::__Cr::char_traits&lt;char&gt; &gt;&amp;&amp;)&#x27;:
          ./../../../../third_party/libc++/src/include/__functional/function.h:450:(.text+0x18): undefined reference to `temporal_rs_OwnedPartialZonedDateTime_from_utf8&#x27;
          /nix/store/rfp8lhk4dl9syfn64rwb3h3c73426p08-binutils-2.44/bin/ld: /build/source/target/x86_64-unknown-linux-gnu/release/deps/libv8-ef6ce659bea65e87.rlib(js-temporal-objects.o): in function `diplomat::result&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt;, temporal_rs::TemporalError&gt; std::__Cr::__function::__policy_func&lt;diplomat::result&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt;, temporal_rs::TemporalError&gt; (std::__Cr::basic_string_view&lt;char16_t, std::__Cr::char_traits&lt;char16_t&gt; &gt;)&gt;::__call_func&lt;v8::internal::temporal::ToTemporalZonedDateTime(v8::internal::Isolate*, v8::internal::DirectHandle&lt;v8::internal::Object&gt;, v8::internal::MaybeDirectHandle&lt;v8::internal::Object&gt;, char const*)::$_1&gt;(std::__Cr::__function::__policy_storage const*, std::__Cr::basic_string_view&lt;char16_t, std::__Cr::char_traits&lt;char16_t&gt; &gt;&amp;&amp;)&#x27;:
          ./../../../../third_party/libc++/src/include/__functional/function.h:450:(.text+0x18): undefined reference to `temporal_rs_OwnedPartialZonedDateTime_from_utf16&#x27;
          collect2: error: ld returned 1 exit status
          
  = note: some `extern` functions couldn&#x27;t be found; some native libraries may need to be installed or have their path specified
  = note: use the `-l` flag to specify native libraries to link
  = note: use the `cargo:rustc-link-lib` directive to specify the native libraries to link with Cargo (see https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-link-lib)

error: could not compile `brioche` (bin &quot;brioche&quot;) due to 1 previous error</pre>
</details>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>brioche</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>brioche</summary>
<pre>          /nix/store/k1dwi3mlxm3vd32r4zg7qj9yi8d4symx-binutils-2.44/bin/ld: /build/source/target/aarch64-unknown-linux-gnu/release/deps/libv8-2cc00af109f7ee3c.rlib(js-temporal-objects.o): in function `temporal_rs::Instant::to_ixdtf_string_with_compiled_data(temporal_rs::TimeZone const*, temporal_rs::ToStringRoundingOptions) const&#x27;:
          ./../../../../third_party/rust/chromium_crates_io/vendor/temporal_capi-v0_0_11/bindings/cpp/temporal_rs/Instant.hpp:171:(.text[_ZNK11temporal_rs7Instant34to_ixdtf_string_with_compiled_dataEPKNS_8TimeZoneENS_23ToStringRoundingOptionsE]+0xe4): undefined reference to `temporal_rs_Instant_to_ixdtf_string_with_compiled_data&#x27;
          /nix/store/k1dwi3mlxm3vd32r4zg7qj9yi8d4symx-binutils-2.44/bin/ld: /build/source/target/aarch64-unknown-linux-gnu/release/deps/libv8-2cc00af109f7ee3c.rlib(js-temporal-objects.o): in function `decltype(auto) std::__Cr::__variant_detail::__visitation::__base::__dispatcher&lt;0ul&gt;::__dispatch&lt;std::__Cr::__variant_detail::__dtor&lt;std::__Cr::__variant_detail::__traits&lt;diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::TimeZone, std::__Cr::default_delete&lt;temporal_rs::TimeZone&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;, (std::__Cr::__variant_detail::_Trait)1&gt;::__destroy()::{lambda(auto:1&amp;)#1}&amp;&amp;, std::__Cr::__variant_detail::__base&lt;(std::__Cr::__variant_detail::_Trait)1, diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::TimeZone, std::__Cr::default_delete&lt;temporal_rs::TimeZone&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;&amp;&gt;(std::__Cr::__variant_detail::__dtor&lt;std::__Cr::__variant_detail::__traits&lt;diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::TimeZone, std::__Cr::default_delete&lt;temporal_rs::TimeZone&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;, (std::__Cr::__variant_detail::_Trait)1&gt;::__destroy()::{lambda(auto:1&amp;)#1}&amp;&amp;, std::__Cr::__variant_detail::__base&lt;(std::__Cr::__variant_detail::_Trait)1, diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::TimeZone, std::__Cr::default_delete&lt;temporal_rs::TimeZone&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;&amp;)&#x27;:
          ./../../../../third_party/libc++/src/include/variant:534:(.text[_ZNSt4__Cr16__variant_detail12__visitation6__base12__dispatcherIJLm0EEE10__dispatchIOZNS0_6__dtorINS0_8__traitsIJN8diplomat2OkINS_10unique_ptrIN11temporal_rs8TimeZoneENS_14default_deleteISC_EEEEEENS8_3ErrINSB_13TemporalErrorEEEEEELNS0_6_TraitE1EE9__destroyEvEUlRT_E_JRNS0_6__baseILSL_1EJSG_SJ_EEEEEEDcSN_DpT0_]+0x10): undefined reference to `temporal_rs_TimeZone_destroy&#x27;
          /nix/store/k1dwi3mlxm3vd32r4zg7qj9yi8d4symx-binutils-2.44/bin/ld: /build/source/target/aarch64-unknown-linux-gnu/release/deps/libv8-2cc00af109f7ee3c.rlib(js-temporal-objects.o): in function `decltype(auto) std::__Cr::__variant_detail::__visitation::__base::__dispatcher&lt;0ul&gt;::__dispatch&lt;std::__Cr::__variant_detail::__dtor&lt;std::__Cr::__variant_detail::__traits&lt;diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;, (std::__Cr::__variant_detail::_Trait)1&gt;::__destroy()::{lambda(auto:1&amp;)#1}&amp;&amp;, std::__Cr::__variant_detail::__base&lt;(std::__Cr::__variant_detail::_Trait)1, diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;&amp;&gt;(std::__Cr::__variant_detail::__dtor&lt;std::__Cr::__variant_detail::__traits&lt;diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;, (std::__Cr::__variant_detail::_Trait)1&gt;::__destroy()::{lambda(auto:1&amp;)#1}&amp;&amp;, std::__Cr::__variant_detail::__base&lt;(std::__Cr::__variant_detail::_Trait)1, diplomat::Ok&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt; &gt;, diplomat::Err&lt;temporal_rs::TemporalError&gt; &gt;&amp;)&#x27;:
          ./../../../../third_party/libc++/src/include/variant:534:(.text[_ZNSt4__Cr16__variant_detail12__visitation6__base12__dispatcherIJLm0EEE10__dispatchIOZNS0_6__dtorINS0_8__traitsIJN8diplomat2OkINS_10unique_ptrIN11temporal_rs25OwnedPartialZonedDateTimeENS_14default_deleteISC_EEEEEENS8_3ErrINSB_13TemporalErrorEEEEEELNS0_6_TraitE1EE9__destroyEvEUlRT_E_JRNS0_6__baseILSL_1EJSG_SJ_EEEEEEDcSN_DpT0_]+0x10): undefined reference to `temporal_rs_OwnedPartialZonedDateTime_destroy&#x27;
          /nix/store/k1dwi3mlxm3vd32r4zg7qj9yi8d4symx-binutils-2.44/bin/ld: /build/source/target/aarch64-unknown-linux-gnu/release/deps/libv8-2cc00af109f7ee3c.rlib(js-temporal-objects.o): in function `v8::internal::temporal::CombinedRecord::operator=(v8::internal::temporal::CombinedRecord&amp;&amp;)&#x27;:
          ./../../../../v8/src/objects/js-temporal-objects.cc:1711:(.text[_ZN2v88internal8temporal14CombinedRecordaSEOS2_]+0xd8): undefined reference to `temporal_rs_TimeZone_destroy&#x27;
          /nix/store/k1dwi3mlxm3vd32r4zg7qj9yi8d4symx-binutils-2.44/bin/ld: ./../../../../v8/src/objects/js-temporal-objects.cc:1711:(.text[_ZN2v88internal8temporal14CombinedRecordaSEOS2_]+0x13c): undefined reference to `temporal_rs_TimeZone_destroy&#x27;
          /nix/store/k1dwi3mlxm3vd32r4zg7qj9yi8d4symx-binutils-2.44/bin/ld: /build/source/target/aarch64-unknown-linux-gnu/release/deps/libv8-2cc00af109f7ee3c.rlib(js-temporal-objects.o): in function `diplomat::result&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt;, temporal_rs::TemporalError&gt; std::__Cr::__function::__policy_func&lt;diplomat::result&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt;, temporal_rs::TemporalError&gt; (std::__Cr::basic_string_view&lt;char, std::__Cr::char_traits&lt;char&gt; &gt;)&gt;::__call_func&lt;v8::internal::temporal::ToTemporalZonedDateTime(v8::internal::Isolate*, v8::internal::DirectHandle&lt;v8::internal::Object&gt;, v8::internal::MaybeDirectHandle&lt;v8::internal::Object&gt;, char const*)::$_0&gt;(std::__Cr::__function::__policy_storage const*, std::__Cr::basic_string_view&lt;char, std::__Cr::char_traits&lt;char&gt; &gt;&amp;&amp;)&#x27;:
          ./../../../../third_party/libc++/src/include/__functional/function.h:450:(.text+0x20): undefined reference to `temporal_rs_OwnedPartialZonedDateTime_from_utf8&#x27;
          /nix/store/k1dwi3mlxm3vd32r4zg7qj9yi8d4symx-binutils-2.44/bin/ld: /build/source/target/aarch64-unknown-linux-gnu/release/deps/libv8-2cc00af109f7ee3c.rlib(js-temporal-objects.o): in function `diplomat::result&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt;, temporal_rs::TemporalError&gt; std::__Cr::__function::__policy_func&lt;diplomat::result&lt;std::__Cr::unique_ptr&lt;temporal_rs::OwnedPartialZonedDateTime, std::__Cr::default_delete&lt;temporal_rs::OwnedPartialZonedDateTime&gt; &gt;, temporal_rs::TemporalError&gt; (std::__Cr::basic_string_view&lt;char16_t, std::__Cr::char_traits&lt;char16_t&gt; &gt;)&gt;::__call_func&lt;v8::internal::temporal::ToTemporalZonedDateTime(v8::internal::Isolate*, v8::internal::DirectHandle&lt;v8::internal::Object&gt;, v8::internal::MaybeDirectHandle&lt;v8::internal::Object&gt;, char const*)::$_1&gt;(std::__Cr::__function::__policy_storage const*, std::__Cr::basic_string_view&lt;char16_t, std::__Cr::char_traits&lt;char16_t&gt; &gt;&amp;&amp;)&#x27;:
          ./../../../../third_party/libc++/src/include/__functional/function.h:450:(.text+0x20): undefined reference to `temporal_rs_OwnedPartialZonedDateTime_from_utf16&#x27;
          collect2: error: ld returned 1 exit status
          
  = note: some `extern` functions couldn&#x27;t be found; some native libraries may need to be installed or have their path specified
  = note: use the `-l` flag to specify native libraries to link
  = note: use the `cargo:rustc-link-lib` directive to specify the native libraries to link with Cargo (see https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-link-lib)

error: could not compile `brioche` (bin &quot;brioche&quot;) due to 1 previous error</pre>
</details>
</details>

---
### `x86_64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>brioche</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>brioche</summary>
<pre>                ...
            &quot;_v8__IdleTask__DELETE&quot;, referenced from:
                _$LT$v8..platform..IdleTask$u20$as$u20$core..ops..drop..Drop$GT$::drop::h9c5115c6780a0822 in libv8-bf85cfba6184fb13.rlib(v8-bf85cfba6184fb13.v8.cc323b016dc6fd75-cgu.14.rcgu.o)
            &quot;_v8__ModuleRequest__GetPhase&quot;, referenced from:
                deno_core::modules::map::ModuleMap::new_module_from_js_source::he28f4ab3267abdc6 (.llvm.18086968936447021984) in libdeno_core-417bb2b063a0ee98.rlib(deno_core-417bb2b063a0ee98.deno_core.fa19e061c4fe0b3d-cgu.03.rcgu.o)
            &quot;_v8__Platform__NewCustomPlatform&quot;, referenced from:
                std::sync::once::Once::call_once::_$u7b$$u7b$closure$u7d$$u7d$::hcefffae5ed8cbe0c (.llvm.3432883436360400971) in libdeno_core-417bb2b063a0ee98.rlib(deno_core-417bb2b063a0ee98.deno_core.fa19e061c4fe0b3d-cgu.10.rcgu.o)
            &quot;_v8__Promise__MarkAsHandled&quot;, referenced from:
                deno_core::modules::map::ModuleMap::mod_evaluate_sync::hb5e1e33129d533e8 in libdeno_core-417bb2b063a0ee98.rlib(deno_core-417bb2b063a0ee98.deno_core.fa19e061c4fe0b3d-cgu.03.rcgu.o)
                deno_core::ops_builtin::op_import_sync::op_import_sync::slow_function_impl::hceec959f52fb29c8 in libdeno_core-417bb2b063a0ee98.rlib(deno_core-417bb2b063a0ee98.deno_core.fa19e061c4fe0b3d-cgu.13.rcgu.o)
            &quot;_v8__Task__DELETE&quot;, referenced from:
                v8::platform::Task::run::h44e5010cc90ff76a in libv8-bf85cfba6184fb13.rlib(v8-bf85cfba6184fb13.v8.cc323b016dc6fd75-cgu.14.rcgu.o)
                _$LT$v8..platform..Task$u20$as$u20$core..ops..drop..Drop$GT$::drop::h7909e4afdfcba72f in libv8-bf85cfba6184fb13.rlib(v8-bf85cfba6184fb13.v8.cc323b016dc6fd75-cgu.14.rcgu.o)
            &quot;_v8__Task__Run&quot;, referenced from:
                v8::platform::Task::run::h44e5010cc90ff76a in libv8-bf85cfba6184fb13.rlib(v8-bf85cfba6184fb13.v8.cc323b016dc6fd75-cgu.14.rcgu.o)
          ld: symbol(s) not found for architecture x86_64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
          

error: could not compile `brioche` (bin &quot;brioche&quot;) due to 1 previous error</pre>
</details>
</details>

---
### `aarch64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>brioche</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>brioche</summary>
<pre>                ...
            &quot;_v8__IdleTask__DELETE&quot;, referenced from:
                _$LT$v8..platform..IdleTask$u20$as$u20$core..ops..drop..Drop$GT$::drop::h663b266641d3fca2 in libv8-70e87352ea199ab7.rlib(v8-70e87352ea199ab7.v8.219aaa42dec884c3-cgu.08.rcgu.o)
            &quot;_v8__ModuleRequest__GetPhase&quot;, referenced from:
                deno_core::modules::map::ModuleMap::new_module_from_js_source::h5c36ceb1826b7c89 (.llvm.16759902339890575089) in libdeno_core-93616ebfe665630c.rlib(deno_core-93616ebfe665630c.deno_core.28f925d309b624c2-cgu.03.rcgu.o)
            &quot;_v8__Platform__NewCustomPlatform&quot;, referenced from:
                std::sync::once::Once::call_once::_$u7b$$u7b$closure$u7d$$u7d$::h4366a0f59d410952 (.llvm.8904444308620169131) in libdeno_core-93616ebfe665630c.rlib(deno_core-93616ebfe665630c.deno_core.28f925d309b624c2-cgu.13.rcgu.o)
            &quot;_v8__Promise__MarkAsHandled&quot;, referenced from:
                deno_core::modules::map::ModuleMap::mod_evaluate_sync::h7e2fd02d8bc7ab2b in libdeno_core-93616ebfe665630c.rlib(deno_core-93616ebfe665630c.deno_core.28f925d309b624c2-cgu.03.rcgu.o)
                deno_core::ops_builtin::op_import_sync::op_import_sync::slow_function_impl::h259d9b819a746d46 in libdeno_core-93616ebfe665630c.rlib(deno_core-93616ebfe665630c.deno_core.28f925d309b624c2-cgu.08.rcgu.o)
            &quot;_v8__Task__DELETE&quot;, referenced from:
                v8::platform::Task::run::h6df04e3caae835ef in libv8-70e87352ea199ab7.rlib(v8-70e87352ea199ab7.v8.219aaa42dec884c3-cgu.08.rcgu.o)
                _$LT$v8..platform..Task$u20$as$u20$core..ops..drop..Drop$GT$::drop::h01223aff1e3d93a8 in libv8-70e87352ea199ab7.rlib(v8-70e87352ea199ab7.v8.219aaa42dec884c3-cgu.08.rcgu.o)
            &quot;_v8__Task__Run&quot;, referenced from:
                v8::platform::Task::run::h6df04e3caae835ef in libv8-70e87352ea199ab7.rlib(v8-70e87352ea199ab7.v8.219aaa42dec884c3-cgu.08.rcgu.o)
          ld: symbol(s) not found for architecture arm64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
          

error: could not compile `brioche` (bin &quot;brioche&quot;) due to 1 previous error</pre>
</details>
</details>

--- a/pkgs/by-name/br/brioche/librusty_v8.nix
+++ b/pkgs/by-name/br/brioche/librusty_v8.nix
@@ -2,12 +2,12 @@
 { fetchLibrustyV8 }:
 
 fetchLibrustyV8 {
-  version = "140.2.0";
+  version = "146.9.0";
   shas = {
     # NOTE; Follows supported platforms of package (see meta.platforms attribute)!
-    x86_64-linux = "0xgkyjx4vhi001p8iv0wqjpal4nl0vj65fnhp5x8ckss6mhanymg";
-    aarch64-linux = "1gal8sylc0k9x37m4lrlncmcfaxn5bdizgi9rhyfb9da9wi41n9f";
-    x86_64-darwin = "0y117lgc1d90f3z285ljazacscs02mmk9b57h403ilgqfly8dzh7";
-    aarch64-darwin = "106n64mrmwk0jzm8zy891kjjbxrr7yggj7m986pxinf8igvab7br";
+    x86_64-linux = "167yh6wabn154axrylam4p76n5a411rqjdgaza4dwsfc5cg2zvv2";
+    aarch64-linux = "1lmpqndmv9lyzsca1sx4prw76fsnk63cr1i15jdmk764j3zyyki6";
+    x86_64-darwin = "1zdlwgq3kgd931ppd9afvqkn45vg8gjsz6k8jb2vnbwzf03grsv0";
+    aarch64-darwin = "0hm2wdkd5palsxdw0r9y5365gdww7f37g5z9260sa639211lipp4";
   };
 }

--- a/pkgs/by-name/br/brioche/package.nix
+++ b/pkgs/by-name/br/brioche/package.nix
@@ -18,16 +18,16 @@
 
 rustPlatform.buildRustPackage (finalAttrs: {
   pname = "brioche";
-  version = "0.1.6";
+  version = "0.1.7";
 
   src = fetchFromGitHub {
     owner = "brioche-dev";
     repo = "brioche";
     tag = "v${finalAttrs.version}";
-    hash = "sha256-tMVBpFCTxesrkj7Z3huVD86baw1tYAtPdPAzYJzTCnA=";
+    hash = "sha256-j0Hi75olubvlHEOzVbW0cMAslZFWCzHiwXaBqmkXzmE=";
   };
 
-  cargoHash = "sha256-LPHJHS69oUvmTaYoMSINCcyYIHYMq++UCUMh9KrCm/0=";
+  cargoHash = "sha256-3UvggE45Gvus5ghd64ZK+Nh/VB3NJBmZNrbStl/xJu0=";
 
   env = {
     OPENSSL_NO_VENDOR = true;
@@ -42,16 +42,8 @@ rustPlatform.buildRustPackage (finalAttrs: {
     openssl
   ];
 
-  checkFlags = [
-    # Require internet access
-    "--skip=test_bake_process"
-  ];
-
-  nativeCheckInputs = [
-    # Otherwise, some tests fail with:
-    # Error: failed to find time zone `Etc/Unknown` since there is no time zone database configured at line 1 column 415
-    tzdata
-  ];
+  # Tests require network access and CA certificates, which are unavailable in the nix sandbox
+  doCheck = false;
 
   nativeInstallCheckInputs = [
     versionCheckHook

